### ISSUE TITLE
[Idea] Consider using flower for celery monitoring

### ISSUE THREAD

**Issue by se7entyse7en** [src-d/sourced-ui#291]
See [link](https://docs.celeryproject.org/en/latest/userguide/monitoring.html#flower-real-time-celery-web-monitor).

As we verified with the demos/pocs celery is an important part of the application that IMO should be monitored. Flower provides a webapp based tool for monitoring and administrating celery. It seems to also provide an http api to expose the functionalities.

As an idea related to this, we could also explore checking the status of the celery queue and eventually manage increase or restart the worker pool for example. In the last demo, for example, a restart of the celery workers would have helped, maybe there's a chance we could automate this kind of issues.

**Comment by rpau** [comment]
It seems that Prometheus could be an alternative.

https://itnext.io/prometheus-monitoring-for-celery-workers-655d089dbd59

**Comment by smacker** [comment]
Yep. I would prefer to use Prometheus to track all the metrics in the same place (celery, webserver, db, cache, ...).

**Comment by se7entyse7en** [comment]
Sure! If we can use it and also extend it to have a centralized view for all the services it would be great!
